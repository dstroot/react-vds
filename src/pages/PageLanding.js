import React, { useEffect } from 'react';
import Masthead from 'Masthead';
import Content from 'Content';
import ContactUs from 'ContactUs';

// React also allows us to create pure, stateless components using a normal
// JavaScript function. A Pure component can replace a component that
// only has a render function. Instead of making a full-blown component
// just to render some content to the screen, we can create a pure one instead.
// Pure components are the simplest, fastest components we can write. They are
// easy to write, simple to reason about.
//
// Advantages to using functional components in React are:
//  - We can do away with the heavy lifting of components, no constructor,
//    state, life-cycle madness, etc.
//  - There is no this keyword (i.e. no need to bind)
//  - Presentational components (also called dumb components) emphasize UI over
//    business logic (i.e. no state manipulation in the component)
//  - Encourages building smaller, self-contained components
//  - Highlights badly written code (for better refactoring)
//  - FAST
//  - They are easy to reuse

const PageLanding = () => {
  // Set the page title using the useEffect hook
  useEffect(() => {
    document.title = `Verite Data Science â€¢ Welcome`;
  });

  return (
    <>
      <Masthead />
      <Content />
      <ContactUs />
    </>
  );
};

export default PageLanding;
