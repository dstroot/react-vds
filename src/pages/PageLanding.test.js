import React from 'react';
import PageLanding from './PageLanding';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';

test('PageLanding renders the landing page', () => {
  const component = renderer.create(
    <Router>
      <PageLanding />
    </Router>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
