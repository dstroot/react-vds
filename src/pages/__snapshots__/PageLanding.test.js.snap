// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLanding renders the landing page 1`] = `
<div>
  <main
    className="bd-masthead"
    id="content"
    role="main"
  >
    <div
      className="container"
    >
      <div
        className="row"
      >
        <div
          className="col-5 mx-auto col-md-5 order-md-2 d-none d-md-block"
        >
          <div
            className="canvas-container"
          >
            <canvas
              className={undefined}
              style={
                Object {
                  "height": "100%",
                  "width": "100%",
                }
              }
            />
          </div>
        </div>
        <div
          className="col-md-7 order-md-1 text-center text-md-left pr-md-5"
        >
          <h1
            className="display-4 mb-3 animated fadeInDown text-primary"
          >
            <img
              alt=""
              className="d-inline-block align-top"
              height="50"
              src="logo.png"
              width="50"
            />
             
            Verite Data Science
          </h1>
          <p
            className="lead text-primary"
          >
            INNOVATION, POWERED BY DATA
          </p>
          <p
            className="lead"
          >
            Our clients demand deep industry knowledge and experience, coupled with innovation and the ability to roll up our sleeves and make business transformation happen. As technology rapidly evolves, it’s more important than ever to have an innovation advisor on your side. Whether it’s time to create a new vision or put one into place, we can help.
          </p>
          <div
            className="d-flex flex-column flex-md-row lead mb-3"
          >
            <a
              className="btn btn-lg btn-outline-primary mt-3 mb-2 mb-md-0 mr-md-2"
              href="/about"
              onClick={[Function]}
            >
              About Verite Data Science
            </a>
          </div>
        </div>
      </div>
    </div>
  </main>
  <div
    className="container"
  >
    <div
      className="card-group my-3"
    >
      <div
        className="card"
      >
        <div
          className="svg-container mx-auto"
        >
          <svg
            viewBox="0 0 96 120"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M60.012 58.98c-0.424 0-0.852-0.136-1.216-0.412l-8.76-6.72-8.444 6.696c-0.864 0.684-2.128 0.54-2.808-0.324-0.688-0.864-0.54-2.12 0.324-2.808l9.664-7.664c0.716-0.572 1.732-0.576 2.464-0.020l10 7.664c0.876 0.676 1.040 1.924 0.368 2.808-0.4 0.512-0.996 0.78-1.592 0.78z"
            />
            <path
              d="M50.012 90.98c-1.104 0-2-0.896-2-2v-36c0-1.104 0.896-2 2-2s2 0.896 2 2v36c0 1.104-0.896 2-2 2z"
            />
            <path
              d="M78 72.98h-17.988c-1.104 0-2-0.896-2-2s0.896-2 2-2h17.988c7.72 0 13.996-6.268 13.996-13.98 0-5.028-2.656-9.556-7.1-12.092-0.664-0.38-1.060-1.112-1-1.88 0.064-0.844 0.1-1.42 0.1-2.012 0-16.54-13.456-30-29.996-30-13.716 0-25.66 9.288-29.044 22.596-0.232 0.916-1.1 1.548-2.028 1.504-0.22-0.012-0.432-0.036-0.648-0.072-10.176-0.072-18.288 8.036-18.288 17.948s8.076 17.976 18 17.976h18.020c1.104 0 2 0.9 2 2 0 1.104-0.896 2.012-2 2.012h-18.012c-12.132 0-22-9.86-22-21.98 0-11.964 9.616-21.728 21.536-21.976 4.356-14.216 17.48-24.004 32.468-24.004 18.744 0 33.996 15.252 33.996 34 0 0.364-0.012 0.728-0.032 1.088 4.98 3.332 8.032 8.928 8.032 14.896-0.004 9.912-8.076 17.976-18 17.976z"
            />
          </svg>
        </div>
        <div
          className="card-body"
        >
          <h4
            className="card-title text-primary"
          >
            Cloud Migration
          </h4>
          <p
            className="card-text"
          >
            Today cloud computing is a given. The next wave of cloud transformtion involves migrating your systems and applications to cloud-native technologies such as containers, and cloud container orchestration with tools like Kubernetes. These technologies avoid cloud infrastructure lock-in and enable the next level of cloud efficiency and agility.
          </p>
        </div>
      </div>
      <div
        className="card"
      >
        <div
          className="svg-container mx-auto"
        >
          <svg
            viewBox="0 0 96 120"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M82.016 77.984h-68a2 2 0 0 1-2-2v-36a2 2 0 0 1 2-2h68c1.1 0 2 .896 2 2v36c0 1.104-.9 2-2 2zm-66-4h64v-32h-64v32z"
            />
            <path
              d="M48.016 93.984H31.052a2 2 0 0 1 0-4c6.756 0 8.944-3.164 8.964-4.068v-7.932c0-1.1.896-2 2-2s2 .9 2 2v7.932c0 1.248-.604 2.724-1.848 4.068h5.848a2 2 0 0 1 0 4z"
            />
            <path
              d="M64.98 93.984H48.016a2 2 0 0 1 0-4h5.848c-1.248-1.344-1.848-2.82-1.848-4.068v-7.932c0-1.1.9-2 2-2s2 .9 2 2v7.932c.016.896 2.208 4.068 8.964 4.068a2 2 0 0 1 0 4zm27.192-32.548c-.316 0-.632-.08-.928-.236a2.002 2.002 0 0 1-.836-2.704 13.915 13.915 0 0 0 1.596-6.492c0-5.036-2.656-9.564-7.096-12.096a1.992 1.992 0 0 1-1.008-1.872c.064-.912.104-1.46.104-2.02 0-16.54-13.456-30-29.996-30-13.716 0-25.66 9.288-29.044 22.596-.232.92-1.084 1.532-2.028 1.508a5.837 5.837 0 0 1-.648-.072h-.064C12.044 30.048 4 38.104 4 47.996c0 3.02.76 6.004 2.196 8.632.532.968.176 2.18-.792 2.716a2.004 2.004 0 0 1-2.716-.792A22.072 22.072 0 0 1 0 47.996C0 36.032 9.616 26.268 21.536 26.02c4.356-14.216 17.48-24.004 32.468-24.004 18.748 0 33.996 15.252 33.996 34 0 .364-.012.728-.032 1.088C92.952 40.432 96 46.028 96 52a17.95 17.95 0 0 1-2.064 8.368 1.965 1.965 0 0 1-1.764 1.068z"
            />
          </svg>
        </div>
        <div
          className="card-body"
        >
          <h4
            className="card-title text-primary"
          >
            Digital Workplace
          </h4>
          <p
            className="card-text"
          >
            Digital transformation isn’t solely driven by technology. The digtal workplace requires giving employees and partners access to the right data, applications and tools from anywhere. We are changing our culture, including where and when we work, to enable enhanced modes of collaboration for our "digital natives".
          </p>
        </div>
      </div>
      <div
        className="card"
      >
        <div
          className="svg-container d-block mx-auto"
        >
          <svg
            viewBox="0 0 32 40"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M25.2 29.3c0-.9-.7-1.6-1.6-1.6h-1.2v-3.5h-13v3.5H8.3c-.9 0-1.6.7-1.6 1.6s.7 1.6 1.6 1.6h15.3c.9 0 1.6-.8 1.6-1.6zm-14.7-4.1h11v2.5h-11v-2.5zm-2.7 4.1c0-.3.3-.6.6-.6h15.3c.3 0 .6.3.6.6s-.3.6-.6.6H8.3c-.3 0-.5-.3-.5-.6zM2 22.1h28c1.1 0 2-.9 2-2v-17c0-1.1-.9-2-2-2H2c-1.1 0-2 .9-2 2v17c0 1.1.9 2 2 2zm28-1H2c-.5 0-1-.5-1-1V17l7.2-7.8 6.2 7.5 10-9.6 6.6 7.8v5.2c0 .6-.5 1-1 1zM2 2.1h28c.5 0 1 .5 1 1v10.2l-6.5-7.7-9.9 9.6-6.3-7.5L1 15.5V3.1c0-.5.5-1 1-1z"
            />
          </svg>
        </div>
        <div
          className="card-body"
        >
          <h4
            className="card-title text-primary"
          >
            Intelligent Analytics
          </h4>
          <p
            className="card-text"
          >
            Data is the currency of business today. A data-powered enterprise integrates analytics and data to generate tangible outcomes by organizing and curating data, while translating it into valuable and actionable insights. It's also about how much data to retain, and how to ensure destruction when it is no longer relevant.
          </p>
        </div>
      </div>
    </div>
  </div>
  <div
    className="position-relative overflow-hidden p-2 p-md-5 m-md-3 text-center"
  >
    <div
      className="col-md-7 p-lg-2 mx-auto my-5"
    >
      <h1
        className="display-4 text-primary"
      >
        Get in Touch
      </h1>
      <p
        className="lead"
      >
        Verite Data Science is providing customized solutions for both startups and Fortune 100 companies. Get in touch to see how we can help.
      </p>
      <div
        className="svg-container mx-auto"
      >
        <svg
          viewBox="0 0 52 65"
          x="0px"
          xmlns="http://www.w3.org/2000/svg"
          y="0px"
        >
          <g>
            <g>
              <path
                d="M19.0400391,28.6206055c-1.7919922,0-3.25-1.4580078-3.25-3.25v-1.7900391c0-1.7919922,1.4580078-3.25,3.25-3.25    c0.8657227,0,1.6811523,0.3378906,2.2958984,0.9511719c0.0004883,0,0.0009766,0.0009766,0.0009766,0.0009766    c0.6147461,0.6142578,0.953125,1.4306641,0.953125,2.2978516v1.7900391    C22.2900391,27.1625977,20.8320313,28.6206055,19.0400391,28.6206055z M19.0400391,22.3305664    c-0.6894531,0-1.25,0.5605469-1.25,1.25v1.7900391c0,0.6894531,0.5605469,1.25,1.25,1.25s1.25-0.5605469,1.25-1.25v-1.7900391    c0-0.3330078-0.1303711-0.6464844-0.3666992-0.8828125C19.6860352,22.4604492,19.3725586,22.3305664,19.0400391,22.3305664z"
              />
            </g>
            <g>
              <path
                d="M49,31.0004883H25.1196289c-0.5522461,0-1-0.4472656-1-1s0.4477539-1,1-1H48v-25H11v12.9697266    c0,0.5527344-0.4477539,1-1,1s-1-0.4472656-1-1V3.0004883c0-0.5527344,0.4477539-1,1-1h39c0.5522461,0,1,0.4472656,1,1v27    C50,30.5532227,49.5522461,31.0004883,49,31.0004883z"
              />
            </g>
            <g>
              <path
                className="st0"
                d="M3.99986,16.9701614v15h21.1199951v-15H3.99986z M12.5798769,25.3701248c0,1.2400513-1,2.25-2.25,2.25    c-0.1099854,0-0.2200317-0.0099487-0.3300171-0.0299683c-1.0899658-0.1600342-1.9199829-1.0900269-1.9199829-2.2200317v-1.789978    c0-1.1300049,0.8300171-2.0599976,1.9199829-2.2199707c0.1099854-0.0200195,0.2200317-0.0300293,0.3300171-0.0300293    c0.6300049,0,1.1900024,0.25,1.5900269,0.6600342c0.4099731,0.4099731,0.6599731,0.9699707,0.6599731,1.5899658V25.3701248z     M21.2898998,25.3701248c0,1.2400513-1.0100098,2.25-2.25,2.25c-1.2400513,0-2.25-1.0099487-2.25-2.25v-1.789978    c0-1.2399902,1.0099487-2.25,2.25-2.25c0.6199951,0,1.1799927,0.25,1.5899658,0.6600342    c0.4100342,0.4099731,0.6600342,0.9699707,0.6600342,1.5899658V25.3701248z"
              />
            </g>
            <g>
              <path
                d="M19.0400391,28.6206055c-1.7919922,0-3.25-1.4580078-3.25-3.25v-1.7900391c0-1.7919922,1.4580078-3.25,3.25-3.25    c0.8657227,0,1.6811523,0.3378906,2.2958984,0.9511719c0.6157227,0.6152344,0.9541016,1.4316406,0.9541016,2.2988281v1.7900391    C22.2900391,27.1625977,20.8320313,28.6206055,19.0400391,28.6206055z M19.0400391,22.3305664    c-0.6894531,0-1.25,0.5605469-1.25,1.25v1.7900391c0,0.6894531,0.5605469,1.25,1.25,1.25s1.25-0.5605469,1.25-1.25v-1.7900391    c0-0.3330078-0.1303711-0.6464844-0.3671875-0.8837891C19.6865234,22.4614258,19.3725586,22.3305664,19.0400391,22.3305664z"
              />
            </g>
            <g>
              <path
                d="M10.3300781,28.6206055c-0.1689453,0-0.3369141-0.015625-0.5048828-0.0458984    c-1.578125-0.2314453-2.7451172-1.5810547-2.7451172-3.2041016v-1.7900391c0-1.6240234,1.1669922-2.9736328,2.7744141-3.2099609    c0.1386719-0.0253906,0.3066406-0.0400391,0.4755859-0.0400391c0.8813477,0,1.699707,0.3408203,2.3051758,0.9599609    c0.6064453,0.6064453,0.9448242,1.4228516,0.9448242,2.2900391v1.7900391    C13.5800781,27.1625977,12.1220703,28.6206055,10.3300781,28.6206055z M10.3300781,22.3305664    c-0.0517578,0-0.1035156,0.0048828-0.1552734,0.0136719c-0.6464844,0.0957031-1.0947266,0.6132813-1.0947266,1.2363281v1.7900391    c0,0.6220703,0.4482422,1.1396484,1.0654297,1.2304688c0.0810547,0.0136719,0.1328125,0.0195313,0.1845703,0.0195313    c0.6894531,0,1.25-0.5605469,1.25-1.25v-1.7900391c0-0.3330078-0.1303711-0.6464844-0.3671875-0.8837891    C10.9790039,22.4575195,10.668457,22.3305664,10.3300781,22.3305664z"
              />
            </g>
            <g>
              <path
                d="M19.0400391,28.6206055c-1.7919922,0-3.25-1.4580078-3.25-3.25v-1.7900391c0-1.7919922,1.4580078-3.25,3.25-3.25    c0.8657227,0,1.6811523,0.3378906,2.2958984,0.9511719c0.6157227,0.6152344,0.9541016,1.4316406,0.9541016,2.2988281v1.7900391    C22.2900391,27.1625977,20.8320313,28.6206055,19.0400391,28.6206055z M19.0400391,22.3305664    c-0.6894531,0-1.25,0.5605469-1.25,1.25v1.7900391c0,0.6894531,0.5605469,1.25,1.25,1.25s1.25-0.5605469,1.25-1.25v-1.7900391    c0-0.3330078-0.1303711-0.6464844-0.3671875-0.8837891C19.6865234,22.4614258,19.3725586,22.3305664,19.0400391,22.3305664z"
              />
            </g>
            <g>
              <path
                d="M21,49.9995117H7c-0.5522461,0-1-0.4472656-1-1v-13c0-0.5527344,0.4477539-1,1-1h14c0.5522461,0,1,0.4472656,1,1v13    C22,49.5522461,21.5522461,49.9995117,21,49.9995117z M8,47.9995117h12v-11H8V47.9995117z"
              />
            </g>
            <g>
              <path
                d="M14,36.9995117c-0.5522461,0-1-0.4472656-1-1v-4c0-0.5527344,0.4477539-1,1-1s1,0.4472656,1,1v4    C15,36.5522461,14.5522461,36.9995117,14,36.9995117z"
              />
            </g>
            <g>
              <path
                d="M21.2646484,37.7875977c-0.0126953,0-0.0249023,0-0.0375977,0    c-0.5512695-0.0019531-1.0297852-0.4511719-1.0283203-1.0019531c0.0009766-0.5517578,0.4155273-0.9980469,0.9667969-0.9980469    h0.065918c0.0092773,0,0.0195313,0,0.0288086,0c2.9150391,0,5.4375-1.9238281,6.1450195-4.6875l1.6259766-6.3486328    c0.137207-0.5351563,0.6831055-0.8564453,1.2167969-0.7207031c0.5351563,0.1367188,0.8579102,0.6816406,0.7207031,1.2167969    l-1.6259766,6.3486328C28.4082031,35.246582,25.0893555,37.7875977,21.2646484,37.7875977z"
              />
            </g>
            <g>
              <path
                d="M3,49.7875977c-0.5522461,0-1-0.4472656-1-1v-6.8339844c0-2.7314453,2.222168-4.9541016,4.9541016-4.9541016    c0.5522461,0,1,0.4472656,1,1s-0.4477539,1-1,1C5.3251953,38.9995117,4,40.324707,4,41.9536133v6.8339844    C4,49.340332,3.5522461,49.7875977,3,49.7875977z"
              />
            </g>
            <g>
              <g>
                <path
                  d="M37.9995117,20.9995117c-3.3085938,0-6-2.6914063-6-6s2.6914063-6,6-6S44,11.690918,44,14.9995117     S41.3081055,20.9995117,37.9995117,20.9995117z M37.9995117,10.9995117c-2.2055664,0-4,1.7939453-4,4s1.7944336,4,4,4     c2.2060547,0,4.0004883-1.7939453,4.0004883-4S40.2055664,10.9995117,37.9995117,10.9995117z"
                />
              </g>
              <g>
                <rect
                  height="2"
                  width="2"
                  x="36.9997559"
                  y="7.9995117"
                />
              </g>
              <g>
                <rect
                  height="1.9997864"
                  transform="matrix(0.7071068 -0.7071068 0.7071068 0.7071068 4.7662187 33.02034)"
                  width="1.9990959"
                  x="41.2426376"
                  y="9.7569427"
                />
              </g>
              <g>
                <rect
                  height="2"
                  width="1.9995117"
                  x="43"
                  y="13.9995117"
                />
              </g>
              <g>
                <rect
                  height="2.0004771"
                  transform="matrix(0.7073509 -0.7068626 0.7068626 0.7073509 -1.2394423 35.4906349)"
                  width="2.0004771"
                  x="41.2419472"
                  y="18.2419491"
                />
              </g>
              <g>
                <rect
                  height="2"
                  width="2"
                  x="36.9995117"
                  y="19.9995117"
                />
              </g>
              <g>
                <rect
                  height="1.9997864"
                  transform="matrix(0.7071068 -0.7071068 0.7071068 0.7071068 -3.7187877 29.5054512)"
                  width="1.9997864"
                  x="32.7569427"
                  y="18.241806"
                />
              </g>
              <g>
                <rect
                  height="2"
                  width="1.9995117"
                  x="31"
                  y="13.9995117"
                />
              </g>
              <g>
                <rect
                  height="1.9994411"
                  transform="matrix(0.7072289 -0.7069846 0.7069846 0.7072289 2.2785254 27.0148869)"
                  width="1.9997864"
                  x="32.7571869"
                  y="9.7566271"
                />
              </g>
            </g>
          </g>
        </svg>
      </div>
      <br />
      <a
        aria-current={null}
        className="btn btn-lg btn-outline-secondary"
        href="/contact"
        onClick={[Function]}
        style={undefined}
      >
        Contact Us
      </a>
    </div>
  </div>
</div>
`;
